webapp/sources/scala-ldap/src/main/scala/com/normation/ldap/sdk/LDAPConnectionProvider.scala

```diff
@@ -389,6 +393,7 @@ class ROPooledSimpleAuthConnectionProvider(
   override val poolSize : Int = 2,
   val blockingModule: Blocking
 ) extends SimpleAuthConnection with PooledConnectionProvider[RoLDAPConnection] {
+  override def poolname: String = s"rudder-ro-auth-max${poolSize}"
```

Comment:
same here ... maybe the first one should have max ... but i don't think max is usefull in the name !

Reply:
i think having a explaination of what the number is would be helpful: rudder-ro-auth-max-size:${poolSize}

Reply:
```
INFO: { "timestamp":"2021-05-20T16:05:10.630Z", "debug-type":"connection-pool", "level":"INFO", "thread-id":93, "thread-name":"zio-default-blocking-5-1815095837", "connection-pool-name":"rudder-auth-ro-size:2", "connection-id":3, "connected-to-address":"localhost", "connected-to-port":1389, "current-available-connections":1, "maximum-available-connections":2, "message":"Checked out an immediately available pooled connection", "ldap-sdk-version":"5.0.1", "ldap-sdk-revision":"3290ee33d4aa17df1aadb4d814d6534375f395a9" }
May 20, 2021 6:05:10 PM com.unboundid.util.Debug log
INFO: { "timestamp":"2021-05-20T16:05:10.656Z", "debug-type":"connection-pool", "level":"INFO", "thread-id":102, "thread-name":"zio-default-blocking-9-1815095837", "connection-pool-name":"rudder-auth-rw-size:2", "connection-id":4, "connected-to-address":"localhost", "connected-to-port":1389, "current-available-connections":2, "maximum-available-connections":2, "message":"Released a connection back to the pool", "ldap-sdk-version":"5.0.1", "ldap-sdk-revision":"3290ee33d4aa17df1aadb4d814d6534375f395a9" }
May 20, 2021 6:05:11 PM com.unboundid.util.Debug log
INFO: { "timestamp":"2021-05-20T16:05:11.025Z", "debug-type":"connection-pool", "level":"INFO", "thread-id":102, "thread-name":"zio-default-blocking-9-1815095837", "connection-pool-name":"rudder-auth-rw-size:2", "connection-id":1, "connected-to-address":"localhost", "connected-to-port":1389, "current-available-connections":1, "maximum-available-connections":2, "message":"Checked out an immediately available pooled connection", "ldap-sdk-version":"5.0.1", "ldap-sdk-revision":"3290ee33d4aa17df1aadb4d814d6534375f395a9" }
``` 
?

Reply:
Actually, there is already a `"maximum-available-connections":2` so the "2" in the name is likely useless.